src/components/Navbar.tsx

```diff
@@ -140,10 +145,11 @@ export default function Navbar() {
 								<Link
 									key={page.id}
 									to={page.href}
-									className={`text-base hover:border-b whitespace-nowrap ${
+									title={page.title}
+									className={`inline-block text-base text-white border-b hover:border-white whitespace-nowrap before:content-[attr(title)] before:block before:h-0 before:overflow-hidden before:invisible before:font-bold ${
 										activePage === page.id
-											? 'border-b border-white font-bold text-white'
-											: 'text-white'
+											? 'border-b border-white font-bold'
```

Comment:
border-b can be outside of the ternary, couldn't it? Because it is always there no matter the resolution.

Reply:
hm, but only for activePage, so i dont know how to write otherwise..